src/core/cwa.ts

```diff
@@ -2,17 +2,23 @@ import * as bluebird from 'bluebird'
 import type { CwaOptions } from '../'
 import Storage from './storage'
 
-const getObject = data => data[0]
-
 export default class Cwa {
   public ctx: any
   public options: CwaOptions
 
+  // Holds the EventSource connection with mercure
+  private eventSource
   private fetcher
   public $storage: Storage
   public $state
 
   constructor (ctx, options) {
+
+    if (process.server) {
```

Comment:
Could add the check `process.env.NODE_ENV !== 'production'`

Reply:
Mhh not really IMO this code should not be committed and is useless in a full docker-environment as tls would work. I need to find a better solution.

Reply:
I've implemented a configuration option that ca only be used in development mode. By default in the demo application I have set allowing unauthorized tls as it won't be run from within a container in development. The template application which uses this module will not have this option set by default.